lib/utils.js

```diff
@@ -30,6 +31,15 @@ marked.use({
   }
 })
 
+const scripts = []
+if (existsSync(path.join(projectDir, 'app', 'assets', 'javascripts', 'application.js'))) {
+  scripts.push('/public/javascripts/application.js')
+}
+if (extensions.legacyGovukFrontendFixesNeeded()) {
+  scripts.push('/extension-assets/govuk-prototype-kit/lib/assets/javascripts/optional/legacy-govuk-frontend-init.js')
+}
+exports.scripts = scripts
```

Comment:
Let's call this `prototypeAppScripts` or something similar to make it super clear what it is.

Reply:
Done